pypeman/channels.py

```diff
@@ -455,6 +457,28 @@ def graph_dot(self, end=''):
         for end, sub in after:
             sub.graph_dot(end=end)
 
+    def jsonable_msg_info_for_admin(self, msg):
```

Comment:
Can you write a test for this method pls ?

Reply:
yes,  will do

Reply:
no tests for this so far, but write tests for serializers:

see: pypeman/tests/test_hlp_serializer.py 6c79af634e196697fb0b9c0418e5fc39e847c7a3